internal/config/5_test_config.go

```diff
@@ -7,4 +7,5 @@ import "github.com/spf13/viper"
 
 func init() {
 	viper.Set("featureFlags.environment", "test")
+	viper.Set("restEndpoints.sources.appId", 11)
```

Comment:
Are we sure that the database ID is stable and same across environments? This does not look right.

Reply:
It is the id of our application at sources service (on stage). It can be changed only when using devel env

Reply:
I'll ask Sources, but I'd love to avoid a request just for the appId.
But if it will need to be, so be it ^_^ We need the config anyway I believe